### DISCUSSION TITLE
How to use inspector in next js 13?

### DISCUSSION THREAD

**Discussion by TheMetaverseEngineer** [brianzinn/react-babylonjs#288] [Q&A] [answered]
Hi, I'm using Next.js 13 and when I use inspector it shows me the UI of the inspector, but the babylon scene is not interactive anymore. I think it is because of this log, which says should render updating: false. How can I fix this?

![image](https://user-images.githubusercontent.com/51158588/235590285-34067dee-68af-4428-b5e6-8fd3940e0a4f.png)

**Comment by brianzinn** [comment]
do you have a repo that you can share?

I'm using an intersection observer with a threshold of zero.  So, it sounds like your canvas isn't visible when it's not rendering: https://developer.mozilla.org/en-US/docs/Web/API/IntersectionObserver/IntersectionObserver

Are you passing a `whenVisibleOnly` in renderOptions?

**Reply by TheMetaverseEngineer** [reply, marked as answer]
Hi, thank you for your quick answer. I was passing `whenVisibleOnly` in render options and that was the problem. Thank u so much.

**Reply by brianzinn** [reply]
was the canvas completely hidden?  it should have kept rendering if even 1 pixel was visible...
